apps/extension/src/background/lib/helpers.ts

```diff
@@ -3,19 +3,17 @@ import { ErrorCode, type RequestID } from "window.ai"
 import type { Err } from "~core/utils/result-monad"
 import { POPUP_HEIGHT, POPUP_WIDTH, RequestInterruptType } from "~core/constants"
 
-async function _requestInterrupt(
-    requestId: RequestID,
```

Comment:
nit: 4 spaces

Reply:
used prettier extension to format this file